### PR TITLE
Add a Dockerfile

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by thatbudakguy** [comment]
is this going to be used to deploy the app?

**Comment by jcoyne** [comment]
Yes

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node:20-alpine
+WORKDIR /usr/app
+COPY package*.json ./
+RUN npm install
```
Comment (thatbudakguy) [inline Dockerfile:4]: maybe `npm ci` so we use `package-lock.json` for the built image?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,6 @@
   "type": "module",
   "scripts": {
     "compile": "tsc",
-    "postinstall": "npm run compile",
```
Comment (thatbudakguy) [inline package.json:9]: README doesn't mention this because it's assumed that install does it, so we should probably add it as an additional step there

**Comment by jcoyne** [comment]
@thatbudakguy thank you for the review. Changes have been made.

**Review by thatbudakguy — APPROVED** [review]

**Pull request merged**